enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/components/Visualization.tsx

```diff
@@ -33,19 +37,23 @@ export const QuestionVisualization = () => {
   const card = question.card();
 
   return (
-    <QueryVisualization
-      className={cx(CS.flexFull, CS.fullWidth, CS.fullHeight)}
-      question={question}
-      rawSeries={[{ card, data: result && result.data }]}
-      isRunning={isQueryRunning}
-      isObjectDetail={false}
-      isResultDirty={false}
-      isNativeEditorOpen={false}
-      result={result}
-      noHeader
-      mode={mode}
-      navigateToNewCardInsideQB={navigateToNewCard}
-      onNavigateBack={onNavigateBack}
-    />
+    <Box w="100%" h="100%" ref={ref}>
+      <Box w={width} h={height}>
```

Comment:
this doesn't look any difference from hard-coding 500px to the container height.
![image](https://github.com/user-attachments/assets/e2c197a7-d6af-443a-9e02-9b2a467fa238)

It seems the child will just expand to whatever the available space the parent has, so without specifying the parent size, you can't measure the actual content size.

This should eliminate nested `Box` and remove the need of `useSdkElementSize` if you just hard-code 500px which I think does the same exact thing.

Reply:
The idea is that this code can also be used for other places where there isn't a NewQuestion element - people will have responsive layouts with flexible divs and the visualization will still be able to fill the space.

In this case, the Tabs component for mantine actually doesn't fill the space given - it just uses the child's size, so it will default to 500px. If you review it now, you'll see that we could make the visualization actually fill the component area like I would expect